tutorials/animation/index.md

```diff
@@ -0,0 +1,874 @@
+    new CurvedAnimation(parent: animation, curve: Curves.easeOut);
+Animation<int> alpha = new IntTween(begin: 0, end: 255).animate(curve);
+{% endprettify %}
+
+You can chain Tweens together using the `.chain()` method,
+in which case a single Animation object will be configured by multiple
+Tween objects called in succession. This is different than calling
+`.animate()` twice, in which case you have two Animate objects,
+each configured with a single Tween.
+
+### Animation notifications
+
+An Animation object can have Listeners and StatusListeners,
+defined with `addListener()` and `addStatusListener()`.
+A Listener is called whenever the value of the animation changes.
+The most common behavior of a Listener is to call `setState()`
+to cause a rebuild. A StatusListener is called when an animation begins,
+ends, moves forward, or moves reverse, as defined by AnimationStatus.
+The next section has an example of the `addListener()` method,
+and [Monitoring the progress of the animation](#monitoring) shows an
+example of `addStatusListener()`.
+
+---
+
+## Animation examples
+
+This section walks you through 5 animation examples.
+Each section provides a link to the source code for that example.
+
+### Rendering animations
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What's the point?</b>
+
+* How to add basic animation to a widget using `addListener()` and
+  `setState()`.
+* Every time the Animation generates a new number, the `addListener()`
+  function calls `setState()`.
+* How to define an AnimatedController with the required `vsync` parameter.
+* Explains the "`..`" syntax in "`..addListener`", also known as Dart's
+  _cascade notation_.
+
+</div>
+
+So far you've learned how to generate a sequence of numbers over time.
+Nothing has been rendered to the screen.
+To render with an Animation&lt;&gt; object, store the Animation object as a
+member of your Widget, then use its value to decide how to draw.
+
+Consider the following application that draws the Flutter logo without
+animation:
+
+<!-- skip -->
+{% prettify dart %}
+import 'package:flutter/material.dart';
+
+class LogoApp extends StatefulWidget {
+  LogoAppState createState() => new LogoAppState();
+}
+
+class LogoAppState extends State<LogoApp> {
+  @override
+  Widget build(BuildContext context) {
+    return new Center(
+      child: new Container(
+        margin: new EdgeInsets.symmetric(vertical: 10.0),
+        height: 300.0,
+        width: 300.0,
+        child: new FlutterLogo(),
+      ),
+    );
+  }
+}
+
+void main() {
+  runApp(new LogoApp());
+}
+{% endprettify %}
+
+The following shows the same code modified to animate the
+logo to grow from nothing to full size. Note that when
+defining an AnimationController, you must pass in a `vsync` object.
+The presence of `vsync` prevents offscreen animations from consuming
```

Comment:
this definition of vsync is accurate. use it in the other place too if you need it in both places. :-)